### PR TITLE
chore: replace dependabot with renovate

### PR DESCRIPTION
**Pull request by st3iny**
I took [nextcloud/mail/renovate.json](https://github.com/nextcloud/mail/blob/main/renovate.json) as a template and adjusted it accordingly.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		"npm",
+		"php",
+		"postcss-loader"
+	],
+	"packageRules": [
+		{
+			"description": "Request JavaScript reviews",
+			"matchManagers": ["npm"],
+			"reviewers": [
+				"@JuliaKirschenheuter",
```
Comment (ChristophWurst) [inline renovate.json:45]:
```suggestion
				"@GretaD",
```

File: renovate.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	"commitMessageAction": "Bump",
+	"commitMessageTopic": "{{depName}}",
+	"commitMessageExtra": "from {{currentVersion}} to {{#if isPinDigest}}{{{newDigestShort}}}{{else}}{{#if isMajor}}{{prettyNewMajor}}{{else}}{{#if isSingleVersion}}{{prettyNewVersion}}{{else}}{{#if newValue}}{{{newValue}}}{{else}}{{{newDigestShort}}}{{/if}}{{/if}}{{/if}}{{/if}}",
+	"rangeStrategy": "bump",
+	"rebaseWhen": "conflicted",
+	"ignoreUnstable": false,
+	"baseBranches": [
+		"main",
+		"stable4.4",
+		"stable4.3"
```
Comment (st3iny) [inline renovate.json:27]: This means updating dependencies all the way down to Nextcloud 25. Is this ok? I noticed in mail we still update stable1.15 which corresponds to 24 (and lower).

@ChristophWurst

Comment (st3iny) [reply renovate.json:27]: AFAIK, this only applies to vulnerability alerts though.

Comment (ChristophWurst) [reply renovate.json:27]: yes only for vulns and even that is broken https://github.com/renovatebot/renovate/discussions/20542#discussioncomment-5389609

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**